pkg/services/folder/service.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 type Service interface {
-	GetFolders(ctx context.Context, user *user.SignedInUser, orgID int64, limit int64, page int64) ([]*models.Folder, error)
```

Comment:
if we remove the GetFolders here, we might able to remove GetFolders from folderfake and change the expectedFolders type to []*folder.Folder